app/models/cas_seeds/match_decision_reasons.rb

```diff
@@ -7,63 +7,66 @@
-      'Ineligible for Housing Program',
-      'Client refused offer',
-      'Self-resolved',
-      'Falsification of documents',
-      'Additional screening criteria imposed by third parties',
-      'Health and Safety',
-    ]
+      ['CORI', nil],
+      ['SORI', nil],
+      ['Immigration status', nil],
+      ['Household did not respond after initial acceptance of match', nil],
+      ['Ineligible for Housing Program', nil],
+      ['Client refused offer', nil],
+      ['Self-resolved', nil],
+      ['Falsification of documents', nil],
+      ['Additional screening criteria imposed by third parties', nil],
+      ['Health and Safety', nil],
+    ].freeze
 
     HSA_PROVIDER_ONLY_REASONS = [
-      'Household could not be located',
-      'Ineligible for Housing Program',
-      'Client refused offer',
-      'Health and Safety',
-    ]
+      ['Household could not be located', nil],
+      ['Ineligible for Housing Program', nil],
+      ['Client refused offer', nil],
+      ['Health and Safety', nil],
+    ].freeze
 
     SHELTER_AGENCY_REASONS = [
-      "Does not agree to services",
-      "Unwilling to live in that neighborhood",
-      "Unwilling to live in SRO",
-      "Does not want housing at this time",
-      "Unsafe environment for this person",
-      "Client has another housing option",
-    ]
+      ['Does not agree to services', nil],
+      ['Unwilling to live in that neighborhood', nil],
+      ['Unwilling to live in SRO', nil],
+      ['Does not want housing at this time', nil],
+      ['Unsafe environment for this person', nil],
+      ['Client has another housing option', nil],
+    ].freeze
 
     SHELTER_AGENCY_NOT_WORKING_WITH_CLIENT_REASONS = [
-      "Barred from working with agency",
-      "Hospitalized",
-      "Don’t know / disappeared",
-      "Incarcerated",
-    ]
+      ['Barred from working with agency', nil],
+      ['Hospitalized', nil],
+      ['Don’t know / disappeared', nil],
+      ['Incarcerated', nil],
+    ].freeze
 
     ADMINISTRATIVE_CANCEL_REASONS = [
-      'Match expired',
-      'Client has declined match',
-      'Client has disengaged',
-      'Client has disappeared',
-      'SSP CORI',
-      'HSP CORI',
-      'Incarcerated',
-      'Vacancy should not have been entered',
-      'Client received another housing opportunity',
-      'Client no longer eligible for match',
-      'Client deceased',
-      'Vacancy filled by other client'
-    ]
+      ['Match expired', nil],
+      ['Client has declined match', nil],
+      ['Client has disengaged', nil],
+      ['Client has disappeared', nil],
+      ['SSP CORI', nil],
+      ['HSP CORI', nil],
+      ['Incarcerated', nil],
+      ['Vacancy should not have been entered', nil],
+      ['Client received another housing opportunity', nil],
+      ['Client no longer eligible for match', nil],
+      ['Client deceased', nil],
+      ['Vacancy filled by other client', nil],
+    ].freeze
```

Comment:
If every one of these is nil, what's the point?  Is this just place-holder until we know what the second values should be?

Reply:
Yes, I figured we would need to have a discussion on what the disposition of each status would be...